src/rpc/methods/chain.rs

```diff
@@ -869,6 +869,58 @@ quickcheck::quickcheck! {
     }
 }
 
+pub async fn get_parent_receipts<DB: Blockstore + Send + Sync + 'static>(
+    data: Ctx<DB>,
+    message_receipts: Cid,
+) -> Result<Vec<ApiReceipt>> {
+    let store = data.state_manager.blockstore();
+
+    let mut receipts = Vec::new();
+
+    // Try Receipt_v4 first. (Receipt_v4 and Receipt_v3 are identical, use v4 here)
+    if let Ok(amt) = Amt::<fvm_shared4::receipt::Receipt, _>::load(&message_receipts, store)
+        .map_err(|_| {
+            ErrorObjectOwned::owned::<()>(
+                1,
```

Comment:
What is this magic number?

Reply:
I don't know 🤡

Reply:
It's an implementation-defined server-error code for `JSONRPC`.